bot/src/infrastructure/cache/redis-client.ts

```
@@ -1,19 +1,141 @@
+  }
+  return redisClient;
+}
 
-redis.on("error", (err) => {
-  logger.error(`Redis client error: ${err.message}`);
-});
+// Extend Redis client with additional methods for chart caching
+class ExtendedRedisClient {
+  constructor() {
+    // Client will be lazily initialized on first use
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)